metricbeat/module/kubernetes/_meta/test/stats_summary.json

```diff
@@ -121,7 +121,11 @@
       "inodesFree": 473551,
       "inodes": 473560,
       "inodesUsed": 9,
-      "name": "default-token-sg8x5"
+      "name": "default-token-sg8x5",
```

Comment:
can you please add a separate sample with the `pvcRef` to keep in the test file both examples - one for persistentvolumeclaim and one for the ephemeral volume?